lib/modules/manager/pep621/processors/uv.ts

```diff
@@ -32,6 +33,38 @@ export class UvProcessor implements PyProjectProcessor {
       ),
     );
 
+    // https://docs.astral.sh/uv/concepts/dependencies/#dependency-sources
+    // Skip sources that are either not yet handled by Renovate (e.g. git), or do not make sense to handle (e.g. path).
+    if (uv.sources) {
+      for (const dep of deps) {
+        if (!dep.depName) {
+          continue;
+        }
+
+        const depSource = uv.sources[dep.depName];
+        if (depSource) {
+          let skipReason: SkipReason | undefined;
+
+          if (depSource.git) {
+            skipReason = 'git-dependency';
+          } else if (depSource.url) {
+            skipReason = 'unsupported-url';
+          } else if (depSource.path) {
+            skipReason = 'path-dependency';
+          } else if (depSource.workspace) {
+            skipReason = 'inherited-dependency';
+          } else {
+            skipReason = 'invalid-dependency-specification';
+          }
+
+          if (skipReason) {
+            dep.currentValue = '';
```

Comment:
I think we usually have the `currentValue` set even if skipping, because it makes it more easy to understand. Any problem if we do that here too?